src/Illuminate/Mail/Transport/SparkPostTransport.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Illuminate\Mail\Transport;
+
+use Swift_Mime_Message;
+use GuzzleHttp\ClientInterface;
+
+class SparkPostTransport extends Transport
+{
+    /**
+     * Guzzle client instance.
+     *
+     * @var \GuzzleHttp\ClientInterface
+     */
+    protected $client;
+
+    /**
+     * The SparkPost API key.
+     *
+     * @var string
+     */
+    protected $key;
+
+    /**
+     * Create a new SparkPost transport instance.
+     *
+     * @param  \GuzzleHttp\ClientInterface  $client
+     * @param  string  $key
+     * @return void
+     */
+    public function __construct(ClientInterface $client, $key)
+    {
+        $this->client = $client;
+        $this->key = $key;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function send(Swift_Mime_Message $message, &$failedRecipients = null)
+    {
+        $this->beforeSendPerformed($message);
+
+        $recipients = $this->getRecipients($message);
+
+        $message->setBcc([]);
```

Comment:
To fix visibile recipients like Mailgun: https://github.com/laravel/framework/pull/10722